step-framework-parent/step-framework-collections/src/test/java/step/core/accessors/AbstractAccessorTest.java

```diff
@@ -191,4 +204,4 @@ public void setProperty1(String property1) {
 		}
 	}
  
-}
\ No newline at end of file
+}

```

Comment:
this empty line at the end is added automatically by someone (most likely my intellij), but all the files have it.